apps/x509.c

```diff
@@ -605,6 +607,9 @@ int x509_main(int argc, char **argv)
         case OPT_CHECKIP:
             checkip = opt_arg();
             break;
+        case OPT_MULTI:
+            multi = 1;
+            break;
```

Comment:
For consistency, better place OPT_MULTI in the same order as used above, i.e., directly after OPT_MODULUS.